### PR TITLE
fix validation exception message

### PR DESCRIPTION
**Pull request by apuzakov**

### CODE REVIEW

File: src/EventListener/ExceptionListener.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         } else {
             switch (substr($code, 0, 1)) {
                 case '4':
-                    $message = 'Input Error';
+                    $message = $exception->getMessage();
```
Comment (kleijnweb) [inline src/EventListener/ExceptionListener.php]: I'm not a fan of exposing exception messages by default. An option could be to allow this conditionally, using an injected flag (which could for example be set to the value of kernel.debug).

Comment (kleijnweb) [inline src/EventListener/ExceptionListener.php]: You're probably looking for this: https://github.com/kleijnweb/swagger-bundle/issues/27. If you check the exception type (should be InvalidParametersException), I guess `getMessage()` would be acceptable, even in `prod`.

**Comment by kleijnweb** [comment]
I think 'instanceof' (and a `use` statement) would work better. You'll have to fix the failing unit test though, and make sure the exception message actually makes sense as output.

**Comment by apuzakov** [comment]
sure. I`ll do it a bit latter

**Comment by kleijnweb** [comment]
Note that the 3.0 is going to have comprehensive validation error feedback, see #51. That'll take some time though, probably a couple of weeks.

If you fix the build checks I will merge this to master and create a new 2.2.x release, if you don't want to wait. Otherwise I will close this.

**Comment by kleijnweb** [comment]
Christmas came early: https://github.com/kleijnweb/swagger-bundle/releases/tag/v3.0.0

**Pull request closed without merging**